examples/near_global_ocean_simulation.jl

```diff
@@ -101,12 +101,11 @@ set!(ocean.model, T=ECCOMetadata(:temperature; dates=date),
 radiation = Radiation(arch)
 
 # The atmospheric data is prescribed using the JRA55 dataset.
-# The number of snapshots that are loaded into memory is determined by
-# the `backend`
-
-# into memory in 41 snapshots at a time. The JRA55 dataset provides atmospheric
+# The JRA55 dataset provides atmospheric
 # data such as temperature, humidity, and wind fields to calculate turbulent fluxes
 # using bulk formulae, see [`CrossRealmFluxes`](@ref).
```

Comment:
The link to `CrossRealmFluxes` which presumably is meant to link to the API reference is broken. Not sure how to fix it.

Reply:
@glwagner If this is not an easy fix, we can leave it for a future PR.

Reply:
Ah I see, I guess the intent is to link to here: https://clima.github.io/ClimaOceanDocumentation/stable/library/internals/#CrossRealFluxes

let me read a bit to see if there is a simple fix

Reply:
I am not 100% sure, but it may be that the module needs to be documented in the "public" interface, eg by adding a section here:

https://github.com/CliMA/ClimaOcean.jl/blob/6d3d8221f760781372631c61fad4fd13e13603dd/docs/src/library/public.md?plain=1#L42

Reply:
Haha, of course. :smile: